.bcr/source.template.json

```diff
@@ -0,0 +1,5 @@
+{
+  "integrity": "",
+  "strip_prefix": "{REPO}-{VERSION}",
+  "url": "https://github.com/{OWNER}/{REPO}/releases/download/{TAG}/capnproto-cpp-{VERSION}.tar.gz"
```

Comment:
The project will need to start using the Github Releases mechanism, and listing archives there, so that [URLs remain stable](https://github.blog/2023-02-21-update-on-the-future-stability-of-source-code-archives-and-hashes/). This should be considered a requirement both for use from `WORKSPACE` and Bzlmod (BCR will reject the module otherwise).